### PR TITLE
[PORT]Particool System - Port of a port of a port

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

This Pr implement a new particle system, the Particool system off Monkestation, which was ported from Mojave sun, Courtesy of [@dwasint](https://github.com/dwasint)!

https://github.com/Monkestation/MonkeStation/pull/685
https://github.com/Monkestation/MonkeStation/pull/698
https://github.com/Monkestation/MonkeStation/pull/703
https://github.com/Monkestation/MonkeStation/pull/708

## Why It's Good For The Game

Brand new particle system, more aesthetics for the station.

## Testing Photographs and Procedure

<details>

https://user-images.githubusercontent.com/75247747/193154981-7a65c95c-c361-4398-846b-fa31229fcf45.mp4

[More to follow soon]

</details>

## Changelog
:cl:Pigeonverde322, Dwasint
add: Added a brand new Particle system, the Particool from Monkestation
add: Added a particle editor and a particle loader system
add: Added the particles to the Bonfire and flares
code: increased the size for the tgui error maximum line length
/:cl:

### CODE REVIEW

**Comment by ivanmixo** [comment]
That campfire looks extremely out of place in my opinion

**Comment by dwasint** [comment]
Love to see it

**Comment by dwasint** [comment]
> That campfire looks extremely out of place in my opinion

i agree but its a good showpiece for what you could do with particles

**Comment by PowerfulBacon** [comment]
Very cool

**Comment by PowerfulBacon** [comment]
The fire effect looks cool, the fire is a tad too intense imo, the embers are fucking awesome though

File: tgui/.eslintrc.yml [hunk 1]
```diff
@@ -369,7 +369,7 @@ rules:
   # max-depth: error
   ## Enforce a maximum line length
   max-len: [error, {
-    code: 80,
+    code: 128,
```
Comment (itsmeow) [inline tgui/.eslintrc.yml:372]: undocumented change

**Review by itsmeow — APPROVED** [review]
looks good, it's basically an entire new code section that doesn't use anything else, so not much to review

**Pull request merged**